api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,18 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// * HTTP :ref:`router filter <config_http_filters_router>`
+// The resource_names_subscribe field in DeltaDiscoveryRequest contains a list of virtual host names or aliases
```

Comment:
I think we need 1-2 sentences explaining what a VHDS is and why you need one :)

Reply:
Added a short explanation